### ISSUE TITLE
S_rarefied in get_mob_stats

### ISSUE THREAD

**Issue by rueuntal** [MoBiodiv/mobr#152]
@FelixMay - there is an issue with rarefied S in `get_mob_stats`, not sure if it's a bug or if it's supposed to work that way. If some plots have zero abundances, and the user doesn't specify `n_rare_samples`, rarefied S will be zero. This can be seen in the invasion data set:
```
data(inv_comm)
data(inv_plot_attr)
inv_mob_in = make_mob_in(inv_comm, inv_plot_attr)
inv_stats = get_mob_stats(inv_mob_in, 'group', ref_group = "uninvaded", nperm=100)
plot(inv_stats)
```

**Comment by FelixMay** [comment]
Hi @rueuntal , that is a bug! However it should be fixed in PR #151. When I run the following code with my latest version
`inv_stats = get_mob_stats(inv_mob_in, group_var = "group", ref_group = "uninvaded", n_perm = 20)`
`plot(inv_stats, index = "S_rare")` 

I get the correct plot. At the sample level n = 5, which is the default minimum, but the minimum can also be changes as argument `n_rare_min` in `get_mob_stats` and at the group level n = 250, which is n at sample scale times the number of samples in treatments. I decided with Jon to link the n at the sample scale and at the group scale, since we assume linear scaling of N. Does that make sense?

Let me know when there are bugs in PR  #151

**Comment by rueuntal** [comment]
Awesome, thanks!
